src/termdict/termdict.rs

```diff
@@ -199,4 +200,11 @@ impl TermDictionary {
     pub fn stream<'a>(&'a self) -> TermStreamer<'a> {
         self.range().into_stream()
     }
+
+    /// Returns a search builder, to stream all of the terms
+    /// within the Automaton
+    pub fn search<'a, A: Automaton>(&'a self, automaton: A) -> TermStreamerBuilder<'a, A> {
+        let sb = self.fst_index.search(automaton);
```

Comment:
Let's avoid riddly short names like `sb`. especially where the type is non-trviail to the reader. 
`sb` => `stream_builder`.